lms/templates/fields/field_readonly_account.underscore

```diff
@@ -0,0 +1,11 @@
+<div class="field">
+    <% if (title) { %>
+        <label class="u-field-title field-label" for="field-label-<%- id %>-default"><%- title %></label>
+    <% } %>
+    <span class="sr" for="u-field-value-<%- id %>"><%- screenReaderTitle %></span>
+    <span class="u-field-value" id="u-field-value-<%- id %>" aria-describedby="u-field-message-help-<%- id %>"><%- value %></span>
```

Comment:
This is a readonly element as shown in screenshot below, So I am going to replace the above `label` tag with a `span` tag.
<img width="723" alt="screen shot 2016-05-16 at 3 36 17 pm" src="https://cloud.githubusercontent.com/assets/5072991/15287397/f798b07c-1b7b-11e6-9344-036f99763085.png">